File: src/main/java/com/indiepost/repository/LinkRepository.java
package com.indiepost.repository;

import com.indiepost.model.analytics.Link;
import org.springframework.data.repository.CrudRepository;

/**
 * Created by jake on 8/9/17.
 */
public interface LinkRepository extends CrudRepository<Link, Long> {

    Link findByUid(String uid);
}

File: src/main/kotlin/com/indiepost/repository/LinkRepository.kt
package com.indiepost.repository

import com.indiepost.model.analytics.Link
import org.springframework.data.repository.CrudRepository

/**
 * Created by jake on 8/9/17.
 */
interface LinkRepository : CrudRepository<Link, Long> {
    fun findByUid(uid: String): Link
}
